collections.yml

```diff
@@ -57,7 +57,6 @@
   name: Development Framework
   name_cn: 开发框架
   items:
-    - android-development-framework # 安卓开发框架
```

Comment:
When you delete android-development-framework, how to handle the projects existed under this item?

Reply:
Two choice:
1. Leave it unchanged until you add android-component-and-projects.yml into collection.yml. 
2. Delete it and adding  android-component-and-projects.yml into collection.yml at the same time.

Reply:
@eyehwan I agree with your choice 1.
@QYFabc is working on tier 2 classes and normally there are a lot of tier 2 classes need be added before its completion, then I can add related tier 1 class. so choice 1 is more flexible for cooprateion.